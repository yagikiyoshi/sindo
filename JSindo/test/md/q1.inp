$external_charges
      1.03614000      1.01377000     -0.06124000     -0.83400000
      0.51044000      0.89445000      0.55141000      0.41700000
      1.99269000      1.19731000      0.49569000      0.41700000
     -0.04323000     -1.01985000     -1.19538000     -0.83400000
      0.43675000     -0.37594000     -0.99732000      0.41700000
     -0.50318000     -0.82514000     -2.09579000      0.41700000
$end
$comment
first qm/mm  calculation in this tutorial
$end

$rem
igdesp                   6
symmetry                off
sym_ignore             true
exchange              hf
basis                 sto-3g
qm_mm                 true
jobtype               force
symmetry              off
sym_ignore            true
print_input           false
qmmm_print            true
scf_convergence       6
$end

$molecule
0 1      
 O           -0.9957200000        0.0160400000        1.2422500000
 H           -1.2542700000       -0.5669700000        1.8472800000
 H           -0.9377900000       -0.4817900000        0.4267500000
$end

