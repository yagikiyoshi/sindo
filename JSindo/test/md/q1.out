
Running Job 1 of 1 q1.inp
qchem q1.inp_27416.0 /scr/yagi/qchem/q1.27370/q1.27370/ 1
/home/mdsoft/qchem/qchem_v4.2/intel/bin/mpi/mpirun_qchem -machinefile /tmp/31363.1.all.q/hostfile -np 1 /home/mdsoft/qchem/qchem_v4.2/intel/exe/qcprog.exe q1.inp_27416.0 /scr/yagi/qchem/q1.27370/q1.27370/

Process 0 of 1 is on diva02.local - thread support 0
initial socket setup ...start
initial socket setup ...done 
now start server 0 ... 
                  Welcome to Q-Chem
     A Quantum Leap Into The Future Of Chemistry


 Q-Chem 4.2, Q-Chem, Inc., Pleasanton, CA (2014).

 Y. Shao,  Z. Gan,  E. Epifanovsky,  A. T. B. Gilbert,  M. Wormit,  
 J. Kussmann,  A. W. Lange,  A. Behn,  J. Deng,  X. Feng,  D. Ghosh,  
 M. Goldey,  P. R. Horn,  L. D. Jacobson,  I. Kaliman,  R. Z. Khaliullin,  
 T. Kus,  A. Landau,  J. Liu,  E. I. Proynov,  Y. M. Rhee,  R. M. Richard,  
 M. A. Rohrdanz,  R. P. Steele,  E. J. Sundstrom,  H. L. Woodcock III,  
 P. M. Zimmerman,  D. Zuev,  B. Albrecht,  E. Alguire,  B. Austin,  
 G. J. O. Beran,  Y. A. Bernard,  E. Berquist,  K. Brandhorst,  
 K. B. Bravaya,  S. T. Brown,  D. Casanova,  C.-M. Chang,  Y. Chen,  
 S. H. Chien,  K. D. Closser,  D. L. Crittenden,  M. Diedenhofen,  
 R. A. DiStasio Jr.,  H. Do,  A. D. Dutoi,  R. G. Edgar,  S. Fatehi,  
 L. Fusti-Molnar,  A. Ghysels,  A. Golubeva-Zadorozhnaya,  J. Gomes,  
 M. W. D. Hanson-Heine,  P. H. P. Harbach,  A. W. Hauser,  E. G. Hohenstein,  
 Z. C. Holden,  T.-C. Jagau,  H. Ji,  B. Kaduk,  K. Khistyaev,  Jaehoon Kim,  
 Jihan Kim,  R. A. King,  P. Klunzinger,  D. Kosenkov,  T. Kowalczyk,  
 C. M. Krauter,  K. U. Lao,  A. Laurent,  K. V. Lawler,  S. V. Levchenko,  
 C. Y. Lin,  F. Liu,  E. Livshits,  R. C. Lochan,  A. Luenser,  P. Manohar,  
 S. F. Manzer,  S.-P. Mao,  N. Mardirossian,  A. V. Marenich,  S. A. Maurer,  
 N. J. Mayhall,  C. M. Oana,  R. Olivares-Amaya,  D. P. O'Neill,  
 J. A. Parkhill,  T. M. Perrine,  R. Peverati,  P. A. Pieniazek,  A. Prociuk,  
 D. R. Rehn,  E. Rosta,  N. J. Russ,  N. Sergueev,  S. M. Sharada,  
 S. Sharma,  D. W. Small,  A. Sodt,  T. Stein,  D. Stuck,  Y.-C. Su,  
 A. J. W. Thom,  T. Tsuchimochi,  L. Vogt,  O. Vydrov,  T. Wang,  
 M. A. Watson,  J. Wenzel,  A. White,  C. F. Williams,  V. Vanovschi,  
 S. Yeganeh,  S. R. Yost,  Z.-Q. You,  I. Y. Zhang,  X. Zhang,  Y. Zhao,  
 B. R. Brooks,  G. K. L. Chan,  D. M. Chipman,  C. J. Cramer,  
 W. A. Goddard III,  M. S. Gordon,  W. J. Hehre,  A. Klamt,  
 H. F. Schaefer III,  M. W. Schmidt,  C. D. Sherrill,  D. G. Truhlar,  
 A. Warshel,  X. Xu,  A. Aspuru-Guzik,  R. Baer,  A. T. Bell,  N. A. Besley,  
 J.-D. Chai,  A. Dreuw,  B. D. Dunietz,  T. R. Furlani,  S. R. Gwaltney,  
 C.-P. Hsu,  Y. Jung,  J. Kong,  D. S. Lambrecht,  W. Liang,  C. Ochsenfeld,  
 V. A. Rassolov,  L. V. Slipchenko,  J. E. Subotnik,  T. Van Voorhis,  
 J. M. Herbert,  A. I. Krylov,  P. M. W. Gill,  M. Head-Gordon

 Contributors to earlier versions of Q-Chem not listed above: 
 R. D. Adamson,  J. Baker,  E. F. C. Byrd,  A. K. Chakraborty,  C.-L. Cheng,  
 H. Dachsel,  R. J. Doerksen,  G. Hawkins,  A. Heyden,  S. Hirata,  
 G. Kedziora,  F. J. Keil,  C. Kelley,  P. P. Korambath,  W. Kurlancheek,  
 A. M. Lee,  M. S. Lee,  D. Liotard,  I. Lotan,  P. E. Maslen,  N. Nair,  
 D. Neuhauser,  R. Olson,  B. Peters,  J. Ritchie,  N. E. Schultz,  
 N. Shenvi,  A. C. Simmonett,  K. S. Thanthiriwatte,  Q. Wu,  W. Zhang

 Q-Chem 4.2.0 for Intel X86 Linux

 Parts of Q-Chem use Armadillo 4.100.2 (Dirt Cruiser).
 http://arma.sourceforge.net/

 Q-Chem begins on Fri Nov  7 15:12:03 2014  

Host: 
0

     Scratch files written to /scr/yagi/qchem/q1.27370/q1.27370//
 Jul1914 29348_jherbert 5004   
 Parallel job on  1  processors
Processing $rem in /home/mdsoft/qchem/qchem_v4.2/intel/config/preferences.
 MEM_TOTAL  2000 
Checking the input file for consistency...	...done.
 ----------------------------------------------------------------
             Standard Nuclear Orientation (Angstroms)
    I     Atom           X                Y                Z
 ----------------------------------------------------------------
    1      O      -0.9957200000     0.0160400000     1.2422500000
    2      H      -1.2542700000    -0.5669700000     1.8472800000
    3      H      -0.9377900000    -0.4817900000     0.4267500000
 ----------------------------------------------------------------
 Nuclear Repulsion Energy =     9.6013452554 hartrees
 There are        5 alpha and        5 beta electrons
 Requested basis set is STO-3G
 There are 4 shells and 7 basis functions

Total QAlloc Memory Limit   2000 MB
Mega-Array Size        61 MB
MEM_STATIC part        62 MB

                       Distance Matrix (Angstroms)
             O (  1)   H (  2)
   H (  2)  0.879096
   H (  3)  0.957199  1.457848
 
 A cutoff of  1.0D-09 yielded     10 shell pairs
 There are        34 function pairs
 Smallest overlap matrix eigenvalue = 3.07E-01

  Scale SEOQF with 1.000000e+00/1.000000e+00/1.000000e-01

 Standard Electronic Orientation quadrupole field applied
 Nucleus-field energy     =     0.0000000024 hartrees
Adding external point charges to one-electron Hamiltonian
 Nucleus-charge energy    =     0.0006858560 hartrees
 Charge-charge energy     =    -0.6800496318 hartrees
 Guess from superposition of atomic densities
 Warning:  Energy on first SCF cycle will be non-variational
 A restricted Hartree-Fock SCF calculation will be
 performed using Pulay DIIS extrapolation
 SCF converges when DIIS error is below 1.0E-06
 using 8 threads for integral computing 
 
 -------------------------------------------------------
 OpenMP Integral Computing Module                       
 Release: version 1.0, May 2013, Q-Chem Inc. Pittsburgh 
 -------------------------------------------------------
 Integral Job Info:
 Integral job number is                      22
 Integral operator is                         1
 short-range coefficients               -999999
 long-range coefficients                -999999
 Omega coefficients                           0
 if combine SR and LR in K              -999999
 Integral screening is                        1
 Integral computing path is                   2
 max size of driver memory is            800000
 size of driver memory is                 45585
 size of scratch memory is               800000
 max col of scratch BK array                289
 max len of scratch array in speh3           22
 max len of scratch index in speh4            8
 max int batch size is                       50
 min int batch size is                       15
 fixed nKL is                                44
 max L of basis functions is                  1
 order of int derivative is                   0
 number of shells is                          4
 number of basis is                           7
 number of cartesian basis is                 7
 number of contracted shell pairs            10
 number of primitive shell pairs             90
 maxK2 (contraction) of shell pair            9
 max number of K2 of shell pair               1
 max number of CS2 of shell pair              6
 max number of PS2 of shell pair             54
 mem total for path MDJ                    6006
 -------------------------------------------------------
         cpu   0.01 wall   0.00
 ---------------------------------------
  Cycle       Energy         DIIS Error
 ---------------------------------------
    1     -75.4720609192      4.72E-01
         cpu   0.01 wall   0.00
    2     -75.6058356163      6.44E-02
         cpu   0.01 wall   0.00
    3     -75.6412245335      6.71E-03
         cpu   0.01 wall   0.00
    4     -75.6415390952      1.47E-03
         cpu   0.00 wall   0.00
    5     -75.6415578858      1.06E-05
         cpu   0.00 wall   0.00
    6     -75.6415578870      2.81E-06
         cpu   0.00 wall   0.00
    7     -75.6415578872      2.50E-07 Convergence criterion met
 ---------------------------------------
 SCF time:  CPU 1.47 s  wall 0.81 s
 SCF   energy in the final basis set = -75.6415578872
 Total energy in the final basis set = -75.6415578872
 Analysis of SCF Wavefunction
 
          Ground-State Mulliken Net Atomic Charges

     Atom                 Charge (a.u.)
  ----------------------------------------
      1 O                    -0.442498
      2 H                     0.220253
      3 H                     0.222245
  ----------------------------------------
  Sum of atomic charges =     0.000000

 Electrostatic potential will be computed using the SCF density
 right. NPoint =            6
  --- DONE ESP SUMMARY ---
 The QM part of the Energy is   -75.6415578871600     
 Calculating analytic gradient of the SCF energy
         cpu   0.02 wall   0.00
 Gradient of Coulomb and Exchange Energy
            1           2           3
    1  -0.5102584   0.6243677  -0.1141094
    2  -2.7854069   1.4821922   1.3032147
    3  -0.5514802  -1.3690877   1.9205679
 Max gradient component =       1.465E-01
 RMS gradient           =       8.662E-02
 Gradient time:  CPU 0.58 s  wall 0.72 s
Archival summary:
1\1\diva02\JobtypUnspecified\HF\STO-3G\12\yagi\FriNov715:12:052014FriNov715:12:052014\0\\#,JobtypUnspecified,HF,STO-3G,\\0,1\O\H,1,0.879096\H,1,0.957199,2,105.025\\HF=-75.6415579\\@

 Total job time:  1.73s(wall), 3.04s(cpu) 
 Fri Nov  7 15:12:05 2014

        *************************************************************
        *                                                           *
        *  Thank you very much for using Q-Chem.  Have a nice day.  *
        *                                                           *
        *************************************************************


0 sent ACK to 0 
now end server 0 ... 
cleanup process ... done
